### PR TITLE
Move Settings out of Masthead into Context Switcher

### PR DESCRIPTION
**Pull request by josemigallas**
**Which issue(s) this PR fixes** 

[THREESCALE-6241: Move "Settings" inside Context Switcher](https://issues.redhat.com/browse/THREESCALE-6241)

Screenshot 2020-10-28 at 10 54 01

**Verification steps** 

1. Account Settings can be accessed from Context Switcher
2. Item gets highlighted inside ContextSwitcher when in Account Settings

### CODE REVIEW

File: app/views/shared/_api_selector.html.slim [hunk 1]
```diff
@@ -1,6 +1,9 @@
 = javascript_pack_tag 'api_selector'
 
+- settings_link = can?(:manage, current_account) ? provider_admin_account_path : edit_provider_admin_user_personal_details_path
```
Comment (damianpm) [inline app/views/shared/_api_selector.html.slim:3]: Is there any reason why `settings_link` is not defined in [menu_helper.rb](https://github.com/3scale/porta/blob/master/app/helpers/menu_helper.rb), as `current_api`, for example?

Comment (josemigallas) [reply app/views/shared/_api_selector.html.slim:3]: Ah, you're right it should be there

File: app/helpers/menu_helper.rb [hunk 2]
```diff
@@ -96,6 +96,12 @@ def audience_link
     end
   end
 
+  def settings_link
+    can?(:manage, current_account)
```
Comment (damianpm) [inline app/helpers/menu_helper.rb:100]: It seems you can't have a multiline ternary in ruby...
Screenshot 2020-10-28 at 15 39 38

Comment (josemigallas) [reply app/helpers/menu_helper.rb:100]: D'oh 🤣

**Review by damianpm — APPROVED** [review]

**Comment by cntlsn** [comment]
looks good to me from screenshot/description!
@josemigallas just to double-check, whatever sub-page of Account settings the user will be, the context switcher will still display "Account settings", correct? (Which is basically how it currently works already)

**Comment by cntlsn** [comment]
looks good to me from screenshot/description!
@josemigallas just to double-check, whatever sub-page of Account settings the user will be, the context switcher will still display "Account settings", correct? (Which is basically how it currently works already)

**Comment by josemigallas** [comment]
> looks good to me from screenshot/description!
> @josemigallas just to double-check, whatever sub-page of Account settings the user will be, the context switcher will still display "Account settings", correct? (Which is basically how it currently works already)

Yes it will and also it will be highlighted in the dropdown. I just tested it again just in case.

**Pull request merged**